platform-coop-toolkit/coop-library-framework
# Add default field set

## Description

Adds fields as per #2 and adds all custom taxonomies.

## Steps to test

1. Create and edit a new resource.
2. Ensure that all default fields listed in #2 can be added and edited.

**Expected behavior:** Fields can be added, saved, and edited.

## Additional information

This fields are editable via Gutenberg blocks in a Gutenberg block template for the resource post type. See [here](https://developer.wordpress.org/block-editor/developers/block-api/block-templates/).

## Related issues
- #2
